crypto/asn1/a_time.c

```diff
@@ -489,7 +489,7 @@ int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
         if (tm->length > 15 && v[14] == '.') {
             f = &v[14];
             f_len = 1;
-            while (14 + f_len < l && isdigit(f[f_len]))
+            while (14 + f_len < l && ossl_isdigit(f[f_len]))
```

Comment:
Looking at the code in more details, non-ASCII comparisons seem appropriate -- this is what the code was originally.  I suggest leaving it pending a deeper investigation.